ui/src/main/webapp/adminTools/webContextPolicyManager/actions.js

```diff
@@ -42,7 +42,7 @@ export const addContextPath = (attribute, binNumber) => (dispatch, getState) =>
   if (!newPath || newPath.trim() === '') { return }
 
   // simple test for invalid path - backend also validates paths as an additional precaution
-  if (!(/^(\/[!#$&-;=?-[\]_a-z~]+)+/.test(newPath))) {
+  if (!(/^(\/[!#$&-;=?-[\]_a-z~]*)+/.test(newPath))) {
```

Comment:
I agree. I'll pull it out as a function with a short comment. I couldn't figure out a better way to validate the context=path portion of a url so basically this is just all the URI-valid characters + a preceding slash